core/src/main/scala/cats/arrow/Arrow.scala

```diff
@@ -6,42 +6,44 @@ import simulacrum.typeclass
 /**
  * Must obey the laws defined in cats.laws.ArrowLaws.
  */
-@typeclass trait Arrow[F[_, _]] extends Category[F] with Strong[F] { self =>
+@typeclass trait Arrow[F[_, _]] extends Category[F] with Strong[F] {
+  self =>
 
   /**
-   * Lift a function into the context of an Arrow.
-   *
-   * In the reference articles "Arrows are Promiscuous...", and in the corresponding Haskell
-   * library `Control.Arrow`, this function is called `arr`.
-   */
+    * Lift a function into the context of an Arrow.
+    *
+    * In the reference articles "Arrows are Promiscuous...", and in the corresponding Haskell
+    * library `Control.Arrow`, this function is called `arr`.
+    */
```

Comment:
I hate to nitpick on this, but this format is unnecessarily changed away from the scala doc convention.

Reply:
Excuse me kailuowang, but I don't understand. What format is changed away? I don't know where I must correct the format, because I don't notice any difference.Please show me better.thx

Reply:
The indentations of the * :)

Reply:
again, it's a total nitpick. I think it's ready to merge.